monkestation/code/modules/storytellers/converted_events/_base_event.dm

```diff
@@ -6,6 +6,9 @@
 	///required number of enemies in roles to exist
 	var/required_enemies = 0
 
+/client
+	var/picking = FALSE
```

Comment:
I suggest making this be a trait on mind instead so we dont need to deal with weird stuff with client DCs